row_generation/helpers.py

```diff
@@ -1,21 +1,25 @@
+""" Helper functions for the row generation module. """
+
 from typing import List
 
 import itertools
 import re
 
-_cross_pn = []
-_lookup_name = ["Intentionally Left Blank", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "E", "T"]
+_CROSS_PN = []
+_LOOKUP_NAME = "!1234567890ET"
 
 
 def convert_pn(pn_str: str) -> List[List[int]]:
+    """ Convert some place notations into a list of rows. """
```

Comment:
```suggestion
    """ Convert a place notation string into a list of places. """
```

Difficult to describe, the implementation is quite tied to `permute()` on RowGenerator. 
Possibly that should become a function here, taking `stage` as an argument instead of `self`? 
A better name than `permute()` could be `apply_place_notation()`

Reply:
I'll rename `permute()` in another PR - this one is pretty much just to get the row_generation folder passing the linter so that we're closer to getting the tests and linting on the CI.